bash_completions.go

```diff
@@ -134,6 +134,9 @@ __handle_flag()
         must_have_one_flag=()
     fi
 
+    # keep flag value with flagname as flaghash
+    flaghash[${flagname}]=${words[ $((c+1)) ]}
```

Comment:
Sorry, I noticed that it is not good. `--flagname foo` will work, but `--flagname=foo` will not work.